Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SurfaceDialTextboxHelper/SurfaceDialTextboxHelperPage.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<Page
```

Comment:
Could you please run xaml styler over this file please (https://visualstudiogallery.msdn.microsoft.com/3de2a3c6-def5-42c4-924d-cc13a29ff5b7)